### PR TITLE
added line 2 (outttofthebox...)

### PR DESCRIPTION
**Pull request by outttofthebox**
I have added some text on line2

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 This line is written on vivekboora/myRepo
+This line is being written on outttofthebox/myRepo
```
Comment (vivekboora) [inline file.txt:2]: its better to change the word 'written' to 'modified'

File: file.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 This line is written on vivekboora/myRepo
```
Comment (vivekboora) [inline file.txt:1]: change the word 'line' to 'text' on first line

File: file.txt (hunk 1 above)
Comment (vivekboora) [inline file.txt:2]: change 'this' to 'that'

**Review by vivekboora — CHANGES_REQUESTED** [review]

**Comment by vivekboora** [comment]
These changes are necessary

**Comment by outttofthebox** [comment]
I did the modifications asked by you

**Comment by vivekboora** [comment]
#1  was an amazing pull request

**Comment by vivekboora** [comment]
#6a018c34d1d93bea5570b64b812999138f4deda5

**Comment by vivekboora** [comment]
commit 6a018c34d1d93bea5570b64b812999138f4deda5

**Comment by vivekboora** [comment]
6a018c3

**Comment by vivekboora** [comment]
```java
for(int i=0 ; i < 5 ; i++)
{
System.out.println("i is : " + i);
}
```

**Comment by vivekboora** [comment]
> Whether 'tis Nobler in the mind to suffer
> The Slings and Arrows of outrageous Fortune,
How big are these slings and in particular, these arrows?

**Comment by vivekboora** [comment]
> Whether 'tis Nobler in the mind to suffer
> The Slings and Arrows of outrageous Fortune,

How big are these slings and in particular, these arrows?

**Comment by vivekboora** [comment]
I :eyes: that :bug: and I :cold_sweat:.
:trophy: for :microscope: it.
:+1: and :sparkles: on this :ship:, it's :fire::poop:!
:clap::tada::panda_face:

**Comment by vivekboora** [comment]
😄 
vivek2702

**Comment by vivekboora** [comment]
nasa-2W-QWAC0mzI-unsplash

**Comment by outttofthebox** [comment]
nice pic

**Pull request merged**